plugins/spark/workflows/spark_pi.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import random
+from operator import add
+
+from six.moves import range
+
+from flytekit.sdk.tasks import spark_task, inputs, outputs, python_task
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input
+
+
+@inputs(partitions=Types.Integer)
+@outputs(out=Types.Float)
+@spark_task(
+    spark_conf={
+        'spark.driver.memory': "1000M",
+        'spark.executor.memory': "1000M",
+        'spark.executor.cores': '1',
+        'spark.executor.instances': '2',
+        'spark.driver.cores': '1',
+        'spark.hadoop.mapred.output.committer.class': "org.apache.hadoop.mapred.DirectFileOutputCommitter",
+        'spark.hadoop.mapreduce.use.directfileoutputcommitter': "true",
+    },
+    cache_version='1')
+def hello_spark(workflow_parameters, spark_context, partitions, out):
+    print("Starting Spark with Partitions: {}".format(partitions))
+
+    n = 100000 * partitions
+    count = spark_context.parallelize(range(1, n + 1), partitions).map(f).reduce(add)
+    pi_val = 4.0 * count / n
+    print("Pi val is :{}".format(pi_val))
+    out.set(pi_val)
+
+
+@inputs(value_to_print=Types.Float, date_triggered=Types.Datetime)
```

Comment:
lets output the value of Pi. And send this output tobe the output of the workflow too